life360/life360_app.groovy

```diff
@@ -91,13 +91,14 @@
 //file:noinspection SpellCheckingInspection
 //file:noinspection unused
 //file:noinspection GroovySillyAssignment
+//file:noinspection GrPackage
 
 import groovy.transform.CompileStatic
 import groovy.transform.Field
 import java.text.SimpleDateFormat
 import com.hubitat.app.ChildDeviceWrapper
 
-@Field static final String appVersion = '3.0.20'  // public version
```

Comment:
I noticed this in the log and was going to say it might be hard to remember to update this every time the app is updated.. too bad there wasn't some way to dynamically fetch it.

At least it's not displayed to the end-user so forgetting to update it in the future won't be a big deal